src/domains/userProjectRole/dto/index.ts

```diff
@@ -1,13 +1,31 @@
+import { type RoleDTO } from '@domains/role/dto';
+import { type ProjectDTO } from '@domains/project/dto';
+
+/**
+ * Data Transfer Object (DTO) representing a user's project role.
+ */
 export class UserProjectRoleDTO {
+  /** Unique identifier of the user's project role. */
   id: string;
+  /** Identifier of the user associated with the project role. */
   userId: string;
+  /** Identifier of the project associated with the project role. */
   projectId: string;
+  /** Identifier of the role associated with the project role. */
   roleId: string;
+  /** Identifier of the user who emitted the project role. */
   userEmitterId: string;
```

Comment:
I don´t understand this. Why is this necessary? I the pm's id for this case?

Reply:
Frontend team asked to include it because in the future they may need it.